src/commands/Tools/create-emoji.ts

```diff
@@ -0,0 +1,43 @@
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { SkyraCommand } from '#lib/structures';
+import { PermissionLevels } from '#lib/types/Enums';
+import { fetch, FetchResultTypes } from '#utils/util';
+import { ApplyOptions } from '@sapphire/decorators';
+import { EmojiRegex } from '@sapphire/discord.js-utilities';
+import { Args } from '@sapphire/framework';
+import { Message } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	aliases: ['add-emoji'],
+	generateDashLessAliases: true,
+	cooldown: 10,
+	permissionLevel: PermissionLevels.Moderator,
+	permissions: ['MANAGE_EMOJIS'],
+	description: LanguageKeys.Commands.Tools.CreateEmojiDescription,
+	extendedHelp: LanguageKeys.Commands.Tools.CreateEmojiExtended
+})
+export class UserCommand extends SkyraCommand {
+	public async run(message: Message, args: SkyraCommand.Args) {
+		const emojiData = await args.pick(UserCommand.emojiResolver);
+
+		try {
+			const emojiBuffer = await fetch(`https://cdn.discordapp.com/emojis/${emojiData.id}.png`, FetchResultTypes.Buffer);
+			const emoji = await message.guild!.emojis.create(emojiBuffer, emojiData.name);
```

Comment:
Can 1 server have multiple emojis with the same name? If not then verify name existance first.